docs/modules/ROOT/pages/how-to-guides/Secure-your-supply-chain/proc_java_dependencies.adoc

```diff
@@ -132,7 +212,7 @@ kubectl label secret jvm-build-service-upload spi.appstudio.redhat.com/upload-se
 
 Once these steps are completed the secret should disappear, and the system is ready to use.
```

Comment:
[can't say "once"](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#o) (for translation reasons) unless you mean "one time." have to use "after" or "when" instead.

["(dis)appear" is discouraged](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#a) too bc it sounds magic-y. 

SR: After you complete these steps, the secret should no longer display and the system is ready for you to use.